src/commands/license.ts

```diff
@@ -42,16 +44,15 @@ export class License extends BaseCommand implements ICommand {
             return;
         }
 
-        const [vehicle, fuelInfo, sightings] = await Promise.all([
+        const [vehicleInfo, fuelInfo, sightings] = await Promise.all([
             VehicleInfo.get(license),
             FuelInfo.get(license),
             Sightings.list(license, this.interaction.guildId),
         ]);
 
-        if (!vehicle) {
+        if (!vehicleInfo) {
             this.reply('Ik kon dat kenteken niet vindn kerol');
-
-            this.insertSighting(license);
+            await this.insertSighting(license, 0);
```

Comment:
Maybe this should be `null`. Or rather, if a license does not exist? Do we really want to insert a sighting?